source/math/FloatMatrix.ooc

```diff
@@ -24,25 +25,26 @@ FloatMatrix : cover {
 	dimensions ::= this _dimensions
 	width ::= this _dimensions width
 	height ::= this _dimensions height
-	elements: Float[]
+	_elements: OwnedBuffer
+	elements ::= this _elements pointer as Float*
```

Comment:
So... a downside to doing things this way, that I just discovered, is that the `get` function that this syntactic sugar creates is called *every time* you access `this elements[]`.

This class has been littered with `this elements[y + x * this height]` in place of `this[x, y]` for performance reasons (by inlining the operator `[]` function and thereby accessing the array directly) and this change just nullified that optimization. Now it looks bad for no good reason, and `OwnedBuffer__OwnedBuffer__getpointer__` just bumped to the 3rd most-called function in the other project. :(